### ISSUE TITLE
xmDNS/DNS-SD

### ISSUE THREAD

**Issue by kralux** [jstasiak/python-zeroconf#154]
Anyone interested in implementing xmDNS/DNS-SD so this can be used for IEEE2030.5 standard and SunSpec Common Smart Inverter Profile?

**Comment by jstasiak** [comment]
I don't think I'm familiar with those, is there some documentation we could have a look at? In general if it fits the project and coexists with the existing code I don't see why not.

**Comment by kralux** [comment]
Yes it should be a nice addition to the project, especially because it is
starting to get used in some solar standards (SunSpec Alliance IEEE2030.5).
Here is some info on xmDNS:
https://tools.ietf.org/html/draft-lynn-dnsext-site-mdns-01

It seems that it should be pretty straightforward for you. Let me know what
you think.
Thank you,

Alex Kral

**Comment by jstasiak** [comment]
After a quick glance it does seem like a useful addition, if anyone volunteers and implements this I believe we'll be happy to accept a pull request.

**Comment by kralux** [comment]
Jakub,

Do you know of anyone who would be interested in doing this?
I could probably pay him/her a few hundred dollars for the work...
Let me know.
Thank you,

Alex
